extractor/src/test/java/org/schabi/newpipe/extractor/services/DefaultStreamExtractorTest.java

```diff
@@ -382,4 +386,20 @@ public void testTags() throws Exception {
     public void testSupportInfo() throws Exception {
         assertEquals(expectedSupportInfo(), extractor().getSupportInfo());
     }
+
+    @Test
+    public void testMetadataInfo() throws Exception {
+        final MetaInfo metaInfo = extractor().getMetaInfo().get(0);
+        final MetaInfo expectedMetaInfo = expectedMetadataInfo().get(0);
```

Comment:
This will fail if there is no expected metadata info.

Reply:
i know. that's what i meant on irc :)